### PR TITLE
Add codeowners so PR's are automatically tagged with reviewers

### PR DESCRIPTION
**Pull request by thomasyu888**
**Problem**
Code reviewers have to be assigned manually per PR.

**Solution**
Add codeowners so PR's are automatically tagged with reviewers

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* @Sage-Bionetworks/fair-data
+* @Sage-Bionetworks/python-client-developers
```
Comment (thomasyu888) [inline .github/CODEOWNERS:2]: If you could provide this team write access to this repo, that would be fantastic!

Comment (linglp) [reply .github/CODEOWNERS:2]: there's no schematic-developer team. I added @Sage-Bionetworks/python-client-developers manually here: 
Screenshot 2024-08-25 at 5 04 18 PM

Comment (linglp) [reply .github/CODEOWNERS:2]: just changed the role to "maintain"

**Review by linglp — APPROVED** [review]
LGTM

**Comment by thomasyu888** [comment]
Closing due to FAIR having their own process for code reviews.

**Pull request closed without merging**